### PR TITLE
Versioning

### PR DESCRIPTION
**Pull request by sarodz**
Got rid of PyScaffold as it was being a pain to deal with, we might do the same for rest

### CODE REVIEW

File: .github/workflows/PR.yml [hunk 1]
```diff
@@ -21,6 +22,4 @@ jobs:
     - name: Install Tox and any other packages
       run: pip install tox
     - name: Run Tox
-      env: 
-        RAPID_API_TOKEN: ${{secrets.RAPID_API_TOKEN}}
```
Comment (keremAtaman) [inline .github/workflows/PR.yml:25]: Just out of curiosity, why did you remove RAPID_API_TOKEN from PR.yaml? Won't we need it for checks?

File: docs/Makefile [hunk 2]
```diff
@@ -1,29 +0,0 @@
-# Makefile for Sphinx documentation
```
Comment (keremAtaman) [inline docs/Makefile:1]: In general, I feel like we should add the docs/* to the .gitignore. That being said though, why delete this one? Was it causing clashes with the versioner? I am asking as I have a feeling removing this may make Sphinx unusable

**Review by keremAtaman — COMMENTED** [review]
Oh wow, you've done quite a lot! I would like to ask a few questions though, just so we can be on the same page. Please see my comments on the files for more details

File: .github/workflows/PR.yml (hunk 1 above)
Comment (sarodz) [reply .github/workflows/PR.yml:25]: We don't need it yet so I removed it, we can add it once we need it

File: docs/Makefile (hunk 2 above)
Comment (sarodz) [reply docs/Makefile:1]: I removed most pyscafflold artifacts from the repo, we can use some of the tools it uses and Sphinx can be one of them. We can add as a ticket for this later if we want to
